setup/react/index.html

```diff
@@ -9,7 +9,7 @@
     <meta property="og:title" content="Lisk">
     <meta property="og:description" name="description" content="Lisk is an all-in-one solution to manage your Lisk ID, access and send LSK tokens, as well as stake for validators.">
     <meta property="og:image" content="https://lisk.com/hub/assets/images/og-logo.png">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com http://devnet-service.liskdev.net wss://*.walletconnect.com">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/ https://api.mapbox.com/; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com wss://*.lisk.com http://devnet-service.liskdev.net:9901 ws://devnet-service.liskdev.net:9901 wss://*.walletconnect.com">
```

Comment:
what about alphanet? can we try more generic `connect-scr` approach?

Reply:
While I can hardcode alphanet, testnet and mainnet or use a pattern, we still need to figure out the rest of the URLs

Reply:
for now there is no clear direction on how to make it generic, hence approving and proceeding further. We can seek for improvements later.